CtaReactiveMaster/Sources/Home/NewsAPIRequest.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  NewsAPIRequest.swift
+//  CtaReactiveMaster
+//
+//  Created by 山根大生 on 2020/12/26.
+//
+
+import Foundation
+
+enum Key {
+    static var newsApi: String {
+        let value: String = "30d06e4f9a934402a204fa89f9d9acfc"
+        return value
+    }
```

Comment:
ちなみに`APIKey.plist`を使わなくてもこれでも全然問題ない
ただこの書き方の場合はこれだけでいいはず
```suggestion
    static let newsApi = "30d06e4f9a934402a204fa89f9d9acfc"
```

Reply:
ごめん、このレポジトリpublicだから不正利用される可能性高いのでやっぱり`APIKey.plist`での対応でお願いします:bow: